pkg/reconciler/v1alpha1/taskrun/validate.go

```diff
@@ -65,7 +65,6 @@ func ValidateResolvedTaskResources(params []v1alpha1.Param, rtr *resources.Resol
 				}
 			}
 		}
-		// TODO: make sure there aren't missing???
```

Comment:
:thinking:

Reply:
hahaha no feel free to remove it, I'm not sure what it is either though I suspect I wrote it :sweat_smile: